frontend/device/input.lua

```diff
@@ -1220,4 +1274,64 @@ function Input:waitEvent(now, deadline)
     end
 end
 
+-- Allow toggling the handling of most every kind of input, except for power management related events.
+function Input:inhibitInput(toggle)
+    if toggle then
+        -- Only handle power management events
+        if not self._key_ev_handler then
+            logger.info("Inhibiting user input")
+            self._key_ev_handler = self.handleKeyBoardEv
+            self.handleKeyBoardEv = self.handlePowerManagementOnlyEv
+        end
+        -- And send everything else to the void
+        if not self._oasis_ev_handler then
+            self._oasis_ev_handler = self.handleOasisOrientationEv
+            self.handleOasisOrientationEv = self.voidEv
+        end
+        if not self._abs_ev_handler then
```

Comment:
Just a question: Are these mutually exclusive? If so an `elsif` might speed up a bit.

Reply:
I went defensive in case some platforms do weird shit with those.

Otherwise, I'd just have checked the basic & common EV_ABS handler ;).

Reply:
(It's also not in a performance critical codepath. Like, at all ;p).